pyproject.toml

```diff
@@ -586,6 +579,8 @@ exclude = [
 ignore = [
   "ANN101", # Missing type annotation for `self` in method
   "ANN102", # Missing type annotation for `cls` in class method
+  "N815",   # mixedCase variable in class scope (constants, method names)
```

Comment:
I don't think this is currently triggered anywhere in the codebase

```suggestion
```

Reply:
yep. This isn't triggered. I am removing this.